cypress/integration/AboutMeAndLocation.feature

```diff
@@ -0,0 +1,49 @@
+Feature: About me and and location
+  As a user
+  I would like to add some about me text and a location
+  So others can get some info about me and my location
+
+  The location and about me are show on the user. Later it will be possible
+  to search for users by location.
+
+  Background:
+    Given I am logged in
+
+  Scenario: I change my name
+    When I visit the "settings" page
+    And I enter "Hansi" as my name
+    And I press "Save"
+    And I can click on my profile picture in the top right corner
+    Then I can see a "Hansi" as my name
```

Comment:
This is verbose. Make it more reader-friendly:
```gherkin
 Background:
    Given I am logged in
    And I am on the "settings" page

  Scenario: I change my name
    When save "Hansi" as my new name
    Then I can see my new name when I click on my profile picture in the top right
```

Reply:
check